libs/langchain-pinecone/src/vectorstores.ts

```diff
@@ -425,7 +425,7 @@ export class PineconeStore extends VectorStore {
         documentsWithScores.push([
           new Document({
             id,
-            pageContent: pageContent.toString(),
+            pageContent: pageContent?.toString(),
```

Comment:
Should this fallback to something like an empty string?

Having this unpopulated breaks the typing doesn't it? At the very least I think it'd cause problems downstream

Reply:
Yeah you're right - but it's still required in the input so I'm surprised it passes typechecking

Reply:
Oh yeah, I see that now...I agree, I would have anticipated the typecheck to fail...

Reply:
I think it's subtly being typed as any in the client